### PR TITLE
product-esb 1116 test fail, port issue fixed

### PR DESCRIPTION
**Pull request by erandacr**

### CODE REVIEW

File: modules/integration/test-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java [hunk 1]
```diff
@@ -82,4 +85,37 @@ public static OMElement getCustomPayload(String symbol) {
...
     }
+
+    /**
+     * method to kill existing servers which are bind to the given port
+     *
+     * @param port
+     */
+    public static void shutdownFailsafe(int port) {
+        try {
+            System.out.println("Method to kill already existing servers in port " + port);
```
Comment (nwnpallewela) [inline modules/integration/test-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java]: These system.out.prints has to be removed

File: modules/integration/test-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java [hunk 2]
```diff
@@ -82,4 +85,37 @@ public static OMElement getCustomPayload(String symbol) {
...
+            if (line != null) {
+                line = line.trim();
+                String processId = line.split(" +")[1];
+                System.out.println("There is already a process using " + port + ", process id is - " + processId);
+                if (processId != null) {
+                    String killStr = "kill -9 " + processId;
+                    System.out.println("kill string to kill the process - " + killStr);
+                    Runtime.getRuntime().exec(killStr);
+
+                    System.out.println(
```
Comment (sumuditha-viraj) [inline modules/integration/test-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java]: There are more System.out.println, which should be removed.

Comment (erandacr) [inline modules/integration/test-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java]: Fixed with https://github.com/wso2/product-esb/pull/598

File: modules/integration/test-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java (hunk 1 above)
Comment (erandacr) [inline modules/integration/test-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java]: Fixed with https://github.com/wso2/product-esb/pull/598

**Pull request merged**